### PR TITLE
Bug: LT-2066 - editing admin team

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**

### CODE REVIEW

File: teams/helpers.py [hunk 1]
```diff
@@ -7,3 +7,7 @@ def get_team_by_pk(pk):
         return Team.objects.get(pk=pk)
     except Team.DoesNotExist:
         raise NotFoundError({"team": "Team not found - " + str(pk)})
+
+
+def get_admin_team():
+    return Team.objects.get(pk="00000000-0000-0000-0000-000000000001")
```
Comment (efahy) [inline teams/helpers.py]: Please create a constant for the pk in `conf/constants.py`

File: teams/views.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ def put(self, request, pk):
         """
         Update a team instance
         """
+        if pk == get_admin_team().id:
```
Comment (efahy) [inline teams/views.py]: Correct me if I'm wrong, but you don't need to go to the DB for this, you can just do `if pk == "000...001"`, right?

File: teams/helpers.py (hunk 1 above)
Comment (efahy) [reply teams/helpers.py]: Also if you decide to keep this method (see other comment about not needing to go to the DB), there's a `get_team_by_pk` method you can call from here

Comment (efahy) [reply teams/helpers.py]: In fact, its just above this one

File: conf/constants.py [hunk 3]
```diff
@@ -32,3 +32,7 @@ class Roles:
     IMMUTABLE_ROLES = [INTERNAL_SUPER_USER_ROLE_ID, EXPORTER_SUPER_USER_ROLE_ID, EXPORTER_DEFAULT_ROLE_ID]
 
     EXPORTER_PRESET_ROLES = [EXPORTER_SUPER_USER_ROLE_ID, EXPORTER_DEFAULT_ROLE_ID]
+
+
+class Teams:
+    ADMIN_TEAM_ID = "00000000-0000-0000-0000-000000000001"
```
Comment (a-bek) [inline conf/constants.py]: This is also in `conf/settings`

Perhaps import the id from there?

**Pull request merged**